code/_onclick/hud/new_player.dm

```diff
@@ -1,4 +1,5 @@
 /datum/hud/new_player
+	inventory_shown = TRUE
```

Comment:
what is going on here

Reply:
parent starts off with this set to false, which in this hud hides the menu immediately
should doc this

Reply:
https://github.com/tgstation/tgstation/pull/76443/commits/e74765b3ea02e603b27f346dc73142c28b13f527